Sources/MongoSwift/EventLoopBoundMongoClient.swift

```diff
@@ -0,0 +1,98 @@
+import NIO
+
+// sourcery: skipSyncExport
+/// An event loop bound MongoDB Client.
+public struct EventLoopBoundMongoClient {
+    /// The underlying `MongoClient`.
+    internal let client: MongoClient
+
+    /// The `EventLoop` bound to this `EventLoopBoundMongoClient`.
```

Comment:
The "binding" is actually happening the other way around: this is the `EventLoop` that this client is bound to.